load-test-data/src/app.ts

```diff
@@ -10,49 +10,83 @@ import * as dotenv from 'dotenv';
 
 dotenv.config();
 
-var dbService: IDatabase;
+var dbService: IDatabaseLoader;
 dbService = new MongoData();
 
+const version = process.env.VERSION;
+
 const holderId = null;
-let inputPath = 'input/'
-let outputPath = 'output/'
+let inputPath = 'input/' + version
+let outputPath = 'output/' + version
 
 inputPath = path.join(__dirname, inputPath);
 outputPath = path.join(__dirname, outputPath);
 
+const isSingleStr = process.env.DATA_IS_SINGLE_DOCUMENT;
+var isSingle = isSingleStr?.toLowerCase() == 'true' ? true : false;
+var isSingle = isSingleStr?.toLowerCase()  == 'false' ? false : true;
```

Comment:
Is the intent here that by default this is true? (i.e. unless explicitly 'false' it is true?), do we need the first comparison to == 'true' then?

Reply:
You are correct it is not needed. The default is intended to be true.